bagel/cli.py

```diff
@@ -440,33 +446,28 @@ def derivatives(
     ):
         existing_subject = existing_subs_dict.get(subject)
 
-        # Get existing imaging sessions for the subject
-        # Note: This dictionary can be empty if only bagel pheno was run
-        existing_sessions_dict = dutil.get_imaging_session_instances(
+        # Note: Dictionary of existing imaging sessions can be empty if only bagel pheno was run
+        existing_sessions_dict = get_imaging_session_instances(
             existing_subject
         )
 
-        for session_name, sub_ses_proc_df in sub_proc_df.groupby(
+        for session, sub_ses_proc_df in sub_proc_df.groupby(
```

Comment:
why did you revert `session_name` -> `session`? When I see `session` I expect `isinstance(Session)`, not `session="ses-somename"`.